packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -42,7 +42,7 @@ const getStatus = new tasks.LambdaInvoke(this, 'Get Job Status', {
   // Pass just the field named "guid" into the Lambda, put the
   // Lambda's result in a field called "status" in the response
   inputPath: '$.guid',
-  outputPath: '$.Payload',
+  outputPath: '$.status',
```

Comment:
"$.status" is later used here https://github.com/aws/aws-cdk/pull/24117/files#diff-c3ac3f3bd0729bf5d5d41f940534332445cf6827fec30c0d7e98693c556e6d9aR65